rust/windows-client/src-tauri/src/client/logging.rs

```diff
@@ -28,3 +37,83 @@ pub(crate) fn setup(log_filter: &str) -> Result<Handles> {
         _reloader: reloader,
     })
 }
+
+#[tauri::command]
+pub(crate) async fn start_stop_log_counting(
+    managed: tauri::State<'_, Managed>,
+    enable: bool,
+) -> StdResult<(), String> {
+    managed
+        .ctlr_tx
+        .send(ControllerRequest::StartStopLogCounting(enable))
+        .await
+        .map_err(|e| e.to_string())
+}
+
+#[tauri::command]
+pub(crate) async fn clear_logs() -> StdResult<(), String> {
+    clear_logs_inner().await.map_err(|e| e.to_string())
+}
+
+#[tauri::command]
+pub(crate) async fn export_logs(managed: tauri::State<'_, Managed>) -> StdResult<(), String> {
+    export_logs_inner(managed.ctlr_tx.clone())
+        .await
+        .map_err(|e| e.to_string())
+}
+
+pub(crate) async fn clear_logs_inner() -> Result<()> {
+    todo!()
+}
+
+/// Pops up the "Save File" dialog
```

Comment:
Does this prompt the user for the filename, or can we use a naming scheme like `connlib-DATETIME.zip` to save them some keystrokes?

Reply:
It prompts the user. I could see about pre-filling the filename, I think there's an option for that.

Reply:
Sounds good, yeah that's what the other clients do

Reply:
It doesn't write the zip file yet, but that's known to be possible. And it defaults to the "My Documents" folder, is that okay? I saw on Android it writes the zips next to the logs, but I thought it might be due to Android's file permissions.

![image](https://github.com/firezone/firezone/assets/13400041/e359ba1a-0a2f-4193-a20d-83bfb094d93a)

Reply:
LGTM